src/contactForm.js

```diff
@@ -469,6 +466,8 @@ class RecrasContactForm {
             } else {
                 window.alert(this.languageHelper.translate('CONTACT_FORM_SUBMIT_FAILED'));
             }
+            submitButton.removeAttribute('disabled');
+            this.loadingIndicatorHide();
```

Comment:
Is there a specific reason to re-enable the button in `contactForm` in a different place than in `booking`? (same goes for `vouchers`)

Reply:
Not really, changed now :)